rs/execution_environment/src/scheduler/round_schedule.rs

```diff
@@ -417,9 +423,6 @@ impl RoundSchedule {
 
             let has_aborted_or_paused_execution =
                 canister.has_aborted_execution() || canister.has_paused_execution();
-            if !has_aborted_or_paused_execution {
-                RoundSchedule::apply_priority_credit(canister);
-            }
```

Comment:
I see two more places where we update accumulated priority in this function:
1.`rs.accumulated_priority += factual.into();`
2. `canister.scheduler_state.accumulated_priority = rs.accumulated_priority;`
Can we move them as well?